### PR TITLE
Add a time/interaction budget scope to the general agreement doc

### PR DESCRIPTION
**Pull request by csharrison**
This is coming from a call for consensus in the Mar 23, 2023 meeting.

### CODE REVIEW

**Comment by csharrison** [comment]
@martinthomson how do you want to reconcile our two patches :) ?

**Review by martinthomson — COMMENTED** [review]
So I think yours is better in that it talks about the general concept of privacy budgeting, which is helpful to include.  Mine only talked about limiting information rate.

How about we take your heading and introductory paragraph, but use the added detail from mine in a subsection?  Then of course, we can work on making my wording less florid or whatever.

**Comment by csharrison** [comment]
@martinthomson please take a look. I removed some of your wording around "Proposals must also limit the release of information to sites when users do not visit those sites" because that statement is IMO stronger than anything we discussed (e.g. this technically removes the ability to send data about example.com to a delegate site that never appears as a first party).

File: design-dimensions/Dimensions-with-General-Agreement.md [hunk 1]
```diff
@@ -24,3 +24,9 @@ We’ve explored three main definitions of privacy:
 3. Differential privacy
 
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should aim to provide differential privacy guarantees.
+
+## Privacy budget scoping
+
+The privacy budget scope denotes a "bucketing" of user information leakage (formally described in terms of the privacy definition) that is exposed in the private measurement designs. Proposals define a particular scope (or scopes), within which a certain amount of information can be disclosed.
+
+The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should _at least_ include a time / interaction component to the budget scope. That is, as the user browsers the web and interacts with sites more, we would expect that the private measurement system will allow more information about that user to be disclosed.
```
Comment (bmayd) [inline design-dimensions/Dimensions-with-General-Agreement.md]: The community group has reached general agreement that it is reasonable to expect the disclosure of information about a user by a measurement system to increase as the user's interactions on the web increase and a _Private Measurement Technical Specification MVP_ should _at least_ include privacy budget time period and data accumulation scopes that regulate the  rate of information disclosure.

Comment (csharrison) [reply design-dimensions/Dimensions-with-General-Agreement.md]: I think I prefer what I have. Your suggestion's first sentence only discusses user interaction, and then brings up a time dimension in the second sentence. From what we discussed in the meeting, I think this is an "and/or" relationship which is in the current PR.

File: design-dimensions/Dimensions-with-General-Agreement.md [hunk 2]
```diff
@@ -24,3 +24,9 @@ We’ve explored three main definitions of privacy:
 3. Differential privacy
 
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should aim to provide differential privacy guarantees.
+
+## Privacy budget scoping
+
+The privacy budget scope denotes a "bucketing" of user information leakage (formally described in terms of the privacy definition) that is exposed in the private measurement designs. Proposals define a particular scope (or scopes), within which a certain amount of information can be disclosed.
```
[the remark this answers is not in the document]
Comment (csharrison) [reply design-dimensions/Dimensions-with-General-Agreement.md:30]: LGTM

File: design-dimensions/Dimensions-with-General-Agreement.md [hunk 3]
```diff
@@ -24,3 +24,15 @@ We’ve explored three main definitions of privacy:
...
 
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should aim to provide differential privacy guarantees.
+
+## Privacy unit / privacy budget scoping
+
+A privacy budget scope denotes a boundary for user data leakage, formally described in terms of a privacy definition, which is allowed by a private measurement design. Proposals define a scope, or scopes, within which a limited maximum amount of data may be disclosed.
+
+### Inclusion of a time / interaction dimension
+
+In order to be useful, measurement systems need to produce a continuous release of information about user activity. Measurement systems that provide absolute limits on information release would rapidly degrade in utility, until at the limit only supporting measurement for brand-new users. Instead, measurement systems must limit the _rate_ of information that is released.
```
Comment (martinthomson) [inline design-dimensions/Dimensions-with-General-Agreement.md:34]: Strike “rapidly”. Argumentative

File: design-dimensions/Dimensions-with-General-Agreement.md (hunk 1 above)
Comment (martinthomson) [reply design-dimensions/Dimensions-with-General-Agreement.md]: Use “or”, not “/“

**Review by martinthomson — APPROVED** [review]
Wfm

**Comment by martinthomson** [comment]
Regarding the release of information to delegates, I get the point but while I recognise the value of being able to delegate, I usually don’t distinguish in that way. What the site does with the information we give it is far less interesting than the fact that it got it

File: design-dimensions/Dimensions-with-General-Agreement.md (hunk 1 above)
Comment (csharrison) [reply design-dimensions/Dimensions-with-General-Agreement.md]: Done (applied @martinthomson 's suggestion)

File: design-dimensions/Dimensions-with-General-Agreement.md (hunk 3 above)
Comment (csharrison) [reply design-dimensions/Dimensions-with-General-Agreement.md]: Done

**Comment by csharrison** [comment]
Thanks. @bmayd will wait for your sign-off before submitting.

**Pull request merged**